pkgs/development/python-modules/ml-dtypes/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, pybind11
+}:
+
+buildPythonPackage rec {
+  pname = "ml-dtypes";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jax-ml";
+    repo = "ml_dtypes";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-eqajWUwylIYsS8gzEaCZLLr+1+34LXWhfKBjuwsEhhI=";
+    fetchSubmodules = true;
```

Comment:
we should use our eigen https://github.com/jax-ml/ml_dtypes/blob/main/.gitmodules

Reply:
Apparently, @samuela has tried but didn't manage to make it work...

Reply:
Indeed ! It seems to work fine when bulding `ml-dtypes`.

Reply:
Can `fetchSubmodules` now be removed?

Reply:
Done !

Reply:
No, actually it doesn't work.
This recent patch invalidates your patch:
https://github.com/jax-ml/ml_dtypes/commit/1bfd097e794413b0d465fa34f2eff0f3828ff521

I tried to adapt it but am not able to.

Reply:
if we cannot get it to build with nixpkgs's Eigen, I propose we leave a comment here explaining the situation

Reply:
Left a comment in the code about that. Let's wait for another PR to eventually tackle this issue.